extensions/cornerstone/src/init.js

```diff
@@ -119,28 +183,25 @@ export default function init({ servicesManager, configuration }) {
           title: 'Enter your annotation',
           value: { label: data ? data.text : '' },
           noCloseButton: true,
-          onClose: () => UIDialogService.dismiss({ id: dialogId }),
```

Comment:
I thought onClose was supposed to be additional logic that would fire if someone closed the dialog. Isn't onClose fired when the dialog is dismissed? Why do we need to pass dismiss to onClose?

Reply:
onClose is a function of the dialog component not the dialog service, i'm passing a call to close the dialog to the dialog component which sets the button event